content/docs/iac/using-pulumi/mcp-server.md

```diff
@@ -0,0 +1,240 @@
+---
+title_tag: "Pulumi MCP Server | AI-Assisted Infrastructure as Code"
+meta_desc: "Learn how to use Pulumi's Model Context Protocol (MCP) server to integrate AI assistants like Cursor, Claude Code, and GitHub Copilot with your Pulumi workflow."
+title: MCP server
+h1: Pulumi Model Context Protocol server
+meta_image: /images/docs/meta-images/docs-meta.png
+weight: 6
+menu:
+    iac:
+        name: MCP server
+        parent: iac-using-pulumi
+        weight: 6
+---
+
+The Pulumi Model Context Protocol (MCP) server enables AI-powered coding assistants to seamlessly interact with Pulumi's CLI and registry, bringing real-time infrastructure knowledge and management directly into your development environment. This integration dramatically reduces context switching and accelerates Infrastructure as Code (IaC) development workflows.
```

Comment:
Let's not focus so much on the registry -- that's more of an implementation detail. Emphasize that this enables you to codify cloud architectures and get diffs for those right in your CLI much in the same way you already do for code changes thanks to Git. It also ensures that as you build cloud apps it's repeatable and following best practices and policies.